### PR TITLE
feat: v1.0.4 pre-release

### PR DESCRIPTION
**Pull request by wei3erHase**

### CODE REVIEW

**Review by piesrtasty — APPROVED** [review]

File: script/Deploy.s.sol [hunk 1]
```diff
@@ -114,21 +114,41 @@ contract DeployMainnet is MainnetParams, Deploy {
...
+  }
 }
 
-contract DeployGoerli is GoerliParams, Deploy {
+contract DeployTestnet is TestnetParams, Deploy {
   function setUp() public virtual {
-    _deployerPk = uint256(vm.envBytes32('OP_GOERLI_DEPLOYER_PK'));
-    chainId = 420;
+    _deployerPk = uint256(vm.envBytes32('OP_SEPOLIA_DEPLOYER_PK'));
+    chainId = 11_155_420;
```
[the remark this answers is not in the document]
Comment (wei3erHase) [reply script/Deploy.s.sol:145]: good question, i think is remanecent of a moment when we used it for the ERC20 implementation, i wonder how they do the signatures w/o inputting chainid, perhaps is read on-chain

[the remark this answers is not in the document]
Comment (wei3erHase) [reply script/Deploy.s.sol:145]: wanna rm?

File: script/TestnetParams.s.sol [hunk 2]
```diff
@@ -58,25 +58,37 @@ abstract contract GoerliParams is Contracts, Params {
...
-      redemptionRateUpperBound: RAY * WAD, // unbounded
-      redemptionRateLowerBound: 1 // unbounded
+      redemptionRateUpperBound: PLUS_950_PERCENT_PER_YEAR, // +950%/yr
+      redemptionRateLowerBound: MINUS_90_PERCENT_PER_YEAR // -90%/yr
     });
 
     _pidControllerParams = IPIDController.PIDControllerParams({
-      perSecondCumulativeLeak: HALF_LIFE_30_DAYS, // 0.999998e27
-      noiseBarrier: WAD, // no noise barrier
+      perSecondCumulativeLeak: 999_999_711_200_000_000_000_000_000, // HALF_LIFE_30_DAYS
```
[the remark this answers is not in the document]
Comment (wei3erHase) [reply script/TestnetParams.s.sol:90]: this is TestnetParams, can change it.
- Mainnet Params has a more precise number, can also round it to 15 zeros
- Testnet Params has the previous one (less precision), just to not make the tests fail

[the remark this answers is not in the document]
Comment (piesrtasty) [reply script/TestnetParams.s.sol:90]: ahhh makes sense! all good

File: script/Deploy.s.sol (hunk 1 above)
[the remark this answers is not in the document]
Comment (piesrtasty) [reply script/Deploy.s.sol:145]: if not needed, yeah

**Review by piesrtasty — APPROVED** [review]
👍

**Pull request merged**